pds_github_util/branches/git_actions.py

```diff
@@ -5,11 +5,16 @@
 from git import Repo
 import github3
 
+logger = logging.getLogger('github3')
```

Comment:
I guess only the last one of the logger assignment is used.

Reply:
@tloubrieu-jpl copy. i just wish i knew how to quiet all that logging traffic!